src/js/index.js

```diff
@@ -24,3 +27,37 @@ $(window).scroll(function() {
 });
 
 //Product tile functionality
+
+
+//Sticky nav
+$(document).ready(function(){
+  $(window).resize(function(){
```

Comment:
I'm interested to know why this is scroll handler is set up inside of a resize handler. It looks like you invoke the resize event on the window after attaching the handler. If you remove the `.resize()` at the the end, I'm guessing your scroll event wont work until you manually resize the window. I would remove all of the resize stuff, I don't think it's necessary.